internal/coreinternal/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: [open-telemetry/collector-approvers]
```

Comment:
This one also looks odd.

Reply:
Same as above: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS#L111
The CODEOWNERS file says:
```
internal/coreinternal/                                   @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
```